runtime-modules/proposals/codex/src/proposal_types/mod.rs

```diff
@@ -348,3 +349,54 @@ impl Default for ProposalsConfigParameters {
         }
     }
 }
+
```

Comment:
I suggest to leave with_grace_and_voting_periods() here and move the exact config types (development and testing) closer to the usage - in the node project probably in the separate file (proposal config) because testing and development grace and voting periods are likely to change more often than the rest of the code here.

Reply:
Thats a good suggestion 👍 will do

Reply:
fixed in 2f8e54bdfb789beb7230b425ce84d8d175545046